modules/pel/peltool/peltool.py

```diff
@@ -212,29 +212,43 @@ def severityBasedExclusion(uh: UserHeader, config: Config) -> bool:
             return False
     return True
 
-def excludePEL(uh: UserHeader, config: Config) -> bool:
+def considerPEL(uh: UserHeader, config: Config) -> bool:
     """
     Evaluates whether a PEL meets criteria based on specified conditions.
-    Returns True if the entry should be Excluded, False otherwise.
+    Returns True if the entry should be Included, False otherwise.
     """
-    if config.critSysTerm and uh.eventSeverity != SeverityValues.critSysTermSeverity.value:
+    isHidden = uh.actionFlags & ActionFlagsValues.hiddenActionFlag.value
+
+    if config.critSysTerm and uh.eventSeverity == SeverityValues.critSysTermSeverity.value:
         return True
-    
-    if config.non_serviceable_only and uh.isServiceable():
+
+    if config.non_serviceable_only and not uh.isServiceable():
         return True
     
-    if config.serviceable_only and not uh.isServiceable():
+    if config.serviceable_only and uh.isServiceable():
+        return True
+
+    if config.severities and not severityBasedExclusion(uh, config):
```

Comment:
I think this loses the defaults of only showing serviceable pels on the -l, -n, -a, etc:

```
$ peltool.py -p . -n
{
    "Number of PELs found": 10
}
$ peltool.py -p . -n -s
{
    "Number of PELs found": 7
}
```